packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -601,10 +614,12 @@ class _SnackBarState extends State<SnackBar> {
     final EdgeInsets margin = widget.margin?.resolve(TextDirection.ltr) ?? snackBarTheme.insetPadding ?? defaults.insetPadding!;
 
     final double snackBarWidth = widget.width ?? MediaQuery.sizeOf(context).width - (margin.left + margin.right);
-    // Action and Icon will overflow to a new line if their width is greater
-    // than one quarter of the total Snack Bar width.
-    final bool actionLineOverflow =
-        actionAndIconWidth / snackBarWidth > 0.25;
+    final double actionOverflowThreshold = widget.actionOverflowThreshold
+      ?? snackBarTheme.actionOverflowThreshold
+      ?? defaults.actionOverflowThreshold
+      ?? 0.25;
```

Comment:
```suggestion
      ?? defaults.actionOverflowThreshold!;
```

Reply:
Righto - added a 0.25 default to M2Defaults instead of coding it here, so it can always be fetched from `defaults.`.